vmm/src/vm.rs

```diff
@@ -503,14 +503,14 @@ pub enum ExitBehaviour {
     Reset = 2,
 }
 
-pub struct Vm<'a> {
+pub struct Vm {
     fd: Arc<VmFd>,
     kernel: File,
     memory: Arc<RwLock<GuestMemoryMmap>>,
     threads: Vec<thread::JoinHandle<()>>,
     devices: DeviceManager,
     cpuid: CpuId,
-    config: &'a VmConfig,
+    config: VmConfig,
```

Comment:
Is there anything that modifies the config? That could cause problems later if we have copies.

Reply:
As you mentioned, nothing should modify it once it's passed to the VM. If there's something, I'll fix it.